runtime/browser/runtime_url_request_context_getter.cc

```diff
@@ -139,7 +159,7 @@ net::URLRequestContext* RuntimeURLRequestContextGetter::GetURLRequestContext() {
             net::DISK_CACHE,
             net::CACHE_BACKEND_DEFAULT,
             cache_path,
-            0,
+            GetDiskCacheSize(),
```

Comment:
Sorry for replying later. It's need take time to implement Chromium browser's PrefStore.
The xwalk::GetUserAgent() in this function is also need re-parsing.
How about optimize those after this PR for these command line.